docs/public-api.asciidoc

```diff
@@ -248,9 +248,45 @@ finally
 ----
 
 
+[float]
+[[api-transaction-set-label]]
+==== `void SetLabel(string key, T value) added[1.7.0,Number and boolean labels require APM Server 6.7]
+
+Labels are used to add *indexed* information to transactions, spans, and errors.
+Indexed means the data is searchable and aggregatable in Elasticsearch.
+Multiple labels can be defined with different key-value pairs.
+
+* Indexed: Yes
+* Elasticsearch type: {ref}/object.html[object]
+* Elasticsearch field: `labels` (previously `context.tags` in <v.7.0)
+
+Label values can be a string, boolean, or number.
+Because labels for a given key are stored in the same place in Elasticsearch, all label values of a given key must have the same data type.
+Multiple data types per key will throw an exception, e.g. `{foo: bar}` and `{foo: 42}`
```

Comment:
It looks like you copied this from the Java Agent, who copied it from the APM Overview. . . and they forgot the period 😅 
```suggestion
Multiple data types per key will throw an exception, e.g. `{foo: bar}` and `{foo: 42}`.
```